core/src/main/scala/filodb.core/memstore/PartKeyLuceneIndex.scala

```diff
@@ -1176,7 +1178,7 @@ class TopKPartIdsCollector(limit: Int) extends Collector with StrictLogging {
   }
 }
 
-class PartIdCollector extends SimpleCollector {
+class PartIdCollector(limit: Int) extends SimpleCollector {
   val result: debox.Buffer[Int] = debox.Buffer.empty[Int]
```

Comment:
Now that we know the limit not sure how pre-initializing to size limit would play. That is using ``ofSize`` instead of ``empty``. With default size of 8 is almost always guaranteed to re allocate the buffer.

Reply:
its hard to fix the size, will defer it to later PRs